x-pack/plugins/alerting/server/rules_client/common/audit_events.ts

```diff
@@ -195,7 +200,12 @@ export function adHocRunAuditEvent({
   error,
 }: AdHocRunAuditEventParams): AuditEvent {
   const doc = savedObject
-    ? `${AD_HOC_RUN_SAVED_OBJECT_TYPE} [id=${savedObject.id}]`
+    ? [
+        `${AD_HOC_RUN_SAVED_OBJECT_TYPE} [id=${savedObject.id}]`,
+        savedObject.name && `${savedObject.name}`,
```

Comment:
Same here: `[name=${savedObject.name}]`

Reply:
It's for add hoc. And name is not rule name, but phrase like this:
```
savedObject: {
            type: AD_HOC_RUN_SAVED_OBJECT_TYPE,
            id,
            name: `backfill for rule "${result.attributes.rule.name}"`,
          },
```
Ying add this line before, but I decided to keep it and add to the doc. (before it was not used somehow)